code/game/objects/items/weapons/weaponry.dm

```diff
@@ -158,4 +158,40 @@ obj/item/weapon/wirerod/attackby(var/obj/item/I, mob/user as mob, params)
 
 
 
-
+/obj/item/weapon/switchblade
+	name = "switchblade"
+	icon_state = "switchblade"
+	desc = "A sharp, concealable, spring-loaded knife."
+	flags = CONDUCT
+	force = 20

```

Comment:
16 is just barely stronger than a welding tool, which at that point makes the item utterly useless to spend influence on.